packages/pill/src/Pill.types.ts

```diff
@@ -0,0 +1,24 @@
+import { AnchorHTMLAttributes } from 'react';
+
+export interface ComponentProps {
+  /**
+   * The content of the Pill
+   */
+  children: React.ReactNode;
+  /**
+   * 	Element before the children.
+   */
+  leftIcon?: React.ReactNode;
+  /**
+   * The component used for the root node. Either a string to use a HTML element or a component
+   */
+  Component?: React.ElementType;
+  /**
+   * The id for testing
+   */
+  dataTestId?: string;
+}
+
+type HtmlElementProps = ComponentProps & AnchorHTMLAttributes<HTMLElement>;
+
+export type PillProps = Partial<HtmlElementProps>;
```

Comment:
Why are all of the props not required?
the children prop it is the mandatory prop of the component.

Reply:
resolved